docs/pages/setup/guides/ssh-key-extensions.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Using Teleport CA with Github.
+description:  How to use Teleport SSH short lived certificates with GitHub.
```

Comment:
After reading this description I still don't know what it means.

Are we using teleport certificates to authenticate with GitHub?

Reply:
I've made this `How to use Teleport SSH short lived certificates with the GitHubs Certificate Authority.` Would this improve clarity?